routes/company-routes.js

```diff
@@ -1,105 +0,0 @@
-const express = require("express");
-const router = express.Router();
-const Review = require("../models/review");
-const User = require("../models/user");
-const Company = require("../models/company");
-const { isLoggedIn, isLoggedOut } = require("../middleware/route-guard");
-const { averagesObject, getOverallReviewScore } = require("../public/javascripts/averages");
-
-// Created an individual company page where you could see reviews left for that company. It require accessing the Company and User collections, sorting and deep-populating the query before calculating the averages and displaying all reviews. 
-
-router.get("/:id", (req, res, next) => {
-  const { id } = req.params;
-  let comp;
-  Company.findById(id)
-    .then((company) => {
-      comp = company;
-      return Review.find({ companyBeingReviewed: company })
-        .sort({createdAt:-1})
-        .populate([{ path: "companyBeingReviewed" }, { path: "owner" }]);
-    })
-    .then((reviews) => {
-      reviews.forEach(review => console.log(review.owner));
-      const averagesObj = averagesObject(reviews);
-      const overall = getOverallReviewScore(reviews);
```

Comment:
Naming again - `overall` isn't descriptive enough I think, `overallReviewScore` would be better